userdata/cloud-config-controller

```diff
@@ -20,6 +20,8 @@ exec /usr/bin/coreos-cloudinit --from-file /var/run/coreos/$USERDATA_FILE
 coreos:
   update:
     reboot-strategy: "off"
+    group: upp-k8s-eu-west-1-prod-controller
```

Comment:
Do we need a separate group for pac or other clusters?

Reply:
Hmm, good point.

Do we have separate userdata files for those?

Reply:
Nope, we've been trying to use the same one, that is generated by kube-aws, and not touch it afterwards

Reply:
Ok, in that case we should be able to use some kind of env variable that we can reference in the template to generate the group name.

Do we already have a `upp` / `pac` env variable available?